### DISCUSSION TITLE
v1.77.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3992] [Announcements]
# v1.77.0
Welcome to release `v1.77.0` of Immich. This release includes many improvements and bug fixes across the server, web, and mobile clients. This will improve the experience of using Immich for you all. We hope you enjoy this release.

----

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Web
* feat(web): face tooltips by @daniele-athome in https://github.com/immich-app/immich/pull/3924
* chore(web): create real anchors around admin sidebar buttons by @daniele-athome in https://github.com/immich-app/immich/pull/3925
* feat(web): Improved assets upload by @Willena in https://github.com/immich-app/immich/pull/3850
* feat(web): skeleton on asset loading by @JasBogans in https://github.com/immich-app/immich/pull/3867
* fix(web): images not loading on search and gallery by @valonsogit in https://github.com/immich-app/immich/pull/3902
* fix(web): sidebar artifact when toggle themes by @JasBogans in https://github.com/immich-app/immich/pull/3955
* fix(web): skeleton loading by @jrasm91 in https://github.com/immich-app/immich/pull/3972
* feat(web): show original uploader in shared album photo details by @mrijke in https://github.com/immich-app/immich/pull/3977

### Machine Learning
* chore(ml): memory optimisations by @mertalev in https://github.com/immich-app/immich/pull/3934
* dev(ml): fixed `docker-compose.dev.yml`, updated locust by @mertalev in https://github.com/immich-app/immich/pull/3951
* fix(ml): model downloading improvements by @mertalev in https://github.com/immich-app/immich/pull/3988

### Server
* refactor(web,server): use feature flags for oauth by @jrasm91 in https://github.com/immich-app/immich/pull/3928
* fix(server): non-nullable `IsOptional` by @mertalev in https://github.com/immich-app/immich/pull/3939
* test(server): server-info e2e tests by @jrasm91 in https://github.com/immich-app/immich/pull/3948
* feat(server): advanced settings for transcoding by @mertalev in https://github.com/immich-app/immich/pull/3775
* feat(server): wide gamut thumbnails by @mertalev in https://github.com/immich-app/immich/pull/3658
* fix(server): lint import order by @jrasm91 in https://github.com/immich-app/immich/pull/3974
* fix(server): await thumbnail generation before returning by @mertalev in https://github.com/immich-app/immich/pull/3975
* feat(server): consider shared albums in getByAssetId by @mrijke in https://github.com/immich-app/immich/pull/3978
* refactor(server): update asset endpoint by @jrasm91 in https://github.com/immich-app/immich/pull/3973

### Mobile
* fix(mobile): map markers not loading with int coordinates by @shalong-tanwen in https://github.com/immich-app/immich/pull/3957
* fix(mobile): curated places taking more size on large screens by @shalong-tanwen in https://github.com/immich-app/immich/pull/3959
* fix(mobile): change password page not navigation properly after password change by @ddshd in https://github.com/immich-app/immich/pull/3967

### Others
* chore: Remove obsolete environment variables by @daniele-athome in https://github.com/immich-app/immich/pull/3926
* docs: bulk upload: Fix "upload directory" instructions by @pentix in https://github.com/immich-app/immich/pull/3942
* chore(deps): bump actions/checkout from 3 to 4 by @dependabot in https://github.com/immich-app/immich/pull/3983

## New Contributors
* @Willena made their first contribution in https://github.com/immich-app/immich/pull/3850
* @JasBogans made their first contribution in https://github.com/immich-app/immich/pull/3867
* @valonsogit made their first contribution in https://github.com/immich-app/immich/pull/3902
* @pentix made their first contribution in https://github.com/immich-app/immich/pull/3942
* @mrijke made their first contribution in https://github.com/immich-app/immich/pull/3978

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.76.1...v1.77.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.77.0'>v1.77.0</a>.</em>

**Comment by lionhoho** [comment]
Thanks for the great work! I got error msg while using sudo docker compose up -d

Error response from daemon: manifest unknown

**Reply by lionhoho** [reply]
It is working now, Thank you!

**Comment by magicedy** [comment]
Is there no iOS app for this release? I couldn't find it on the Apple App Store.

**Reply by alextran1502** [reply]
Correct, I am on vacation and don't have access to the phone number Apple sent the code to 😅
